pyproject.toml

```diff
@@ -99,12 +99,17 @@ rosa = []
 #     Tornado 6.
 #     tornado
 # ]
+rose-edit = [
+    "pygobject>=3.50.0",
```

Comment:
This totally failed to pip install for me, but it worked when I conda installed `pygobject`.

Failure with pip:

```
Dependency 'gobject-introspection-1.0' is required but not found.
```

Reply:
That is to be expected.

It's the same as trying to `pip install pygraphviz` if you don't have `graphviz` itself installed. If you are installing from `pip`, then you need to satisfy the system dependencies yourself, otherwise pip-installation will likely fail.